source/slang/slang-ir-diff-jvp.cpp

```diff
@@ -2518,4 +2518,1954 @@ void stripAutoDiffDecorations(IRModule* module)
 }
 
 
+
```

Comment:
We probably want to move this to its own file.

Reply:
Yeah I can do some refactor after getting some basic functionality working.